bcgov/HOUS-permit-portal
# Bphh 2270 update requirement template logic to include a part 3 step code block

## Description

Adds ability to add part 3 as a new requirement to the blocks.

## What type of PR is this? (check all applicable)

- [ x] 🍕 Feature

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-2270

## Steps to QA

1. Go to requirements library
2. Create a part 3 requirement block that has the part 3 requirement
3. Add it to a 4+ new construction permit template and save it
4. Try to click teh start button via the tool.  The file upload option should not have a hot 2000 attached to it.
